src/content/structured/components/section-container/guidance.mdx

```diff
@@ -0,0 +1,48 @@
+---
+path: "/components/section-container"
+
+navPriority: 17
+
+date: "2022-12-19"
+
+title: "Section container"
+
+status: "BETA"
+
+subTitle: "The section container component is used to consistently align an app's content."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/section-container/guidance.mdx"
+
+hidden: false
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/section-container" },
+    { title: "Code", path: "/components/section-container/code" },
+  ]
+---
+
+import usageExample from "./images/fig1-container-usage-example.png";
+
+## Introduction
+
+The section container is used by multiple ICDS components to align content more easily.
+
+The components currently using the section container are:
+
+- [Footer](/components/footer)
+- [Hero](/components/hero)
+- [Page header](/components/page-header)
+- [Top navigation](/components/top-nav)
+
+## Usage
+
+When creating a custom component or a web page, a section container can be used to align and group
+related content to make the end result easier to understand and digest for the end user.
+
+<DoDontCaution
+  imageSrc={usageExample}
```

Comment:
This image doesn't follow our image design guidelines and doesn't explain effectively how the section container can be used in different situations.